common/k8sinstall/util.go

```diff
@@ -35,7 +35,6 @@ func installTheProduct() error {
 	_ = CreateNamespace(common.NSOpenEBS())
 
 	e2eCfg := e2e_config.GetConfig()
-	logf.Log.Info("e2e_config.GetConfig()", "CONFIG", e2eCfg)
```

Comment:
Is this intentionally removed?

Reply:
Perhaps the config should be written to a file (the same file always) so that it is available for post mortem analysis.

Reply:
> Perhaps the config should be written to a file (the same file always) so that it is available for post mortem analysis.

yeah, it can be done 
May be in upcoming PR ?